fixtures/pages-functions-app/tests/index.test.ts

```diff
@@ -206,4 +206,22 @@ describe("Pages Functions", () => {
 			expect(getObject.version).toEqual(object.version);
 		});
 	});
+
+	describe("it still attaches redirects correctly", () => {
```

Comment:
since this is the describe block, maybe make that "it can attach redirects correctly", or smth along those lines.